airbyte-api-server/src/main/java/io/airbyte/api/server/repositories/ConnectionsRepository.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.api.server.repositories;
+
+import io.airbyte.public_api.server.model.generated.Connection;
+import io.micronaut.data.repository.GenericRepository;
+import java.util.UUID;
+
+public interface ConnectionsRepository extends GenericRepository<Connection, UUID> {
```

Comment:
We should probably just delete this class and implementation.  The more I thought about it, we really shouldn't have the public API talk directly to the database for any reason -- it should go through the server/cloud server for everything.

Reply:
+1